about.md

```diff
@@ -7,19 +7,25 @@ title: About
 
 <h1>{{ title }}</h1>
   <p class="subtitle">
-    Servo aims to provide an <strong>independent, modular, embeddable web rendering engine</strong>, allowing developers to deliver content and applications using web standards.
+    Servo aims to empower developers with a <strong>lightweight, high-performance alternative for embedding web technologies</strong> in applications.
   </p>
 <br>
 
-Created by Mozilla Research in 2012, the Servo project is a research and development effort. Stewardship of Servo moved from Mozilla Research to the [**Linux Foundation**](https://www.linuxfoundation.org/) in 2020, where its mission remains unchanged. In 2023 the project moved to [**Linux Foundation Europe**](https://linuxfoundation.eu/).
+Created by Mozilla Research in 2012, the Servo project was started as a research and development effort. Stewardship of Servo moved from Mozilla Research to the [**Linux Foundation**](https://www.linuxfoundation.org/) in 2020, where its mission remains unchanged. In 2023 the project moved to [**Linux Foundation Europe**](https://linuxfoundation.eu/) trying to regain interest from a broader set of the industry.
 
 Servo is written in [**Rust**](https://www.rust-lang.org/), taking advantage of the memory safety properties and concurrency features of the language.
 
-Since its creation in 2012, Servo has contributed to W3C and WHATWG web standards, reporting specification issues and submitting new cross-browser automated tests, and core team members have co-edited new standards that have been adopted by other browsers. As a result, the Servo project helps drive the entire web platform forward, while building on a platform of reusable, modular technologies that implement web standards.
+Servo is an **independent** project hosted at Linux Foundation Europe and managed openly by the [Technical Steering Committee](https://github.com/servo/project/blob/main/governance/README.md).
+
+We are currently working on **transitioning Servo from a research project to a production-ready web engine** by delivering stable APIs, improving user interaction capabilities, and fostering an active community that contributes to its growth.
```

Comment:
```suggestion
The current short-term focus of the project is to **transition Servo from a research project to a production-ready web engine** by delivering stable APIs, improving user interaction capabilities, and fostering an active community that contributes to its growth.
```

It might be better to rewrite this paragraph and the next one in the passive voice to match the rest of the document.